lib/omniauth/strategies/rpi/hydra1.rb

```diff
@@ -0,0 +1,51 @@
+require 'omniauth-oauth2'
+require 'jwt'
+
+module OmniAuth::Strategies::Rpi
+  class Hydra1 < OmniAuth::Strategies::OAuth2
+    option :client_options,
+           :site          => 'https://auth.raspberrypi.org',
```

Comment:
I suspect this will need updating for Hydra v1.0

Reply:
These are defaults, which are overwritten in `profile`